src/main/resources/messages/Mainframer.properties

```diff
@@ -6,6 +6,7 @@ configure.downloader.title=Downloading file
 configure.downloader.text=Downloading {0}...
 configure.fetching.versions.description=Downloading Mainframer versions
 configure.action.name=Configure for Project
+configure.templater.dialog.title=Configure for Project
```

Comment:
I decided to choose `Configure rsync settings` because word `template` does not explain usage of such configuration